src/api/views/payment/checkout_session_view.py

```
@@ -0,0 +1,89 @@
+import datetime
+
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.parsers import JSONParser
+from rest_framework.permissions import AllowAny
+
+from src.api.models import LicencePlate
+from src.api.models.garages.price import Price
+from src.api.serializers import LicencePlateSerializer
+from src.api.serializers.payment.checkout_serializer import CheckoutSessionSerializer
+from src.api.views.payment.checkout_preview_view import _get_prices_to_pay
+from src.core.views import BackendResponse, _OriginAPIView
+from src.users.models import User
+
+from django.shortcuts import redirect
+
+from django.utils import timezone
+
+import stripe
+
+# Set your secret key. Remember to switch to your live secret key in production.
+# See your keys here: https://dashboard.stripe.com/apikeys
+WEBSITE_URL = 'https://po3backend.ddns.net/app'
+
+
+class CreateCheckoutSessionView(_OriginAPIView):
+    """
+    A view to create a payment session
+    """
+
+    permission_classes = [AllowAny]
+    origins = ["web", "app"]
+
+    def get(self, request: Request, format=None) -> BackendResponse:
+        if (resp := super().post(request, format)) is not None:
+            return resp
+        checkout_data = {'licence_plate': request.query_params['licence_plate']}
+        checkout_serializer = CheckoutSessionSerializer(data=checkout_data)
+
+        if checkout_serializer.is_valid():
+            try:
+                licence_plate = LicencePlate.objects.get(user=request.user,
+                                                         licence_plate=checkout_serializer.validated_data[
+                                                             'licence_plate'])
+            except:
```

Idem as above, try using a specific Exception, otherwise other exceptions which you not forsee happening are silently ignored.